tas-client/src/tas-client/FeatureProvider/TasApiFeatureProvider.ts

```diff
@@ -4,12 +4,14 @@
  *--------------------------------------------------------------------------------------------*/
 
 import { IExperimentationFilterProvider } from '../../contracts/IExperimentationFilterProvider';
-import { AxiosResponse } from 'axios';
+import { AxiosResponse, AxiosError } from 'axios';
 import { AxiosHttpClient } from '../Util/AxiosHttpClient';
 import { IExperimentationTelemetry } from '../../contracts/IExperimentationTelemetry';
 import { FilteredFeatureProvider } from './FilteredFeatureProvider';
 import { FeatureData, ConfigData } from './IFeatureProvider';
 
+export const TASAPI_FETCHERROR_EVENTNAME = 'call-tas-error';
```

Comment:
nit: does this need to be exported?

Reply:
I'm exporting it just to be able to access the variable in unit tests.